lib/model/database.dart

```diff
@@ -200,7 +235,29 @@ class AppDatabase extends _$AppDatabase {
       // assume there wasn't also the legacy app before that.
       await m.database.update(schema.globalSettings).write(
         RawValuesInsertable({'legacy_upgrade_state': Constant('noLegacy')}));
-    }
+    },
+    from9To10: (m, schema) async {
+      await m.createTable(schema.intGlobalSettings);
+
+      // To provide a smooth experience for users when they first install a new
+      // version of the app with support for the "last visited account" feature,
+      // we set the first available account as the last visited one. This way,
+      // the user is still taken straight to the first account, just as they
+      // were used to before, instead of being shown the "choose account" page.
+      final firstAccount = await (m.database.select(schema.accounts)
+        ..limit(1)).getSingleOrNull();
+      if (firstAccount == null) return;
+
+      final firstAccountId = firstAccount.read<int>('id');
+      await m.database.into(schema.intGlobalSettings).insert(
```

Comment:
Actually, though, let's have a higher-level comment instead:
```suggestion
      // Like `globalStore.setLastVisitedAccount(firstAccountId)`,
      // as of the schema at the time of this migration.
      await m.database.into(schema.intGlobalSettings).insert(
```

Then I think the one below isn't needed.